marklogic-data-hub-central/ui/src/components/detail-header/detail-header.tsx

```diff
@@ -46,7 +46,7 @@ const DetailHeader: React.FC<Props> = (props) => {
           Object.keys(props.document.envelope.instance).forEach(instance => {
             if (instance !== 'info') {
               Object.keys(props.document.envelope.instance[instance]).forEach(function (key) {
-                if (props.primaryKey === props.document.envelope.instance[instance][key]) {
+                if (props.primaryKey == props.document.envelope.instance[instance][key]) {
```

Comment:
Is this necessary to get this to work? Triple equal is better for strict equality checks.

Reply:
They are not the same datatype.